src/transformers/remove_collinear.ts

```diff
@@ -0,0 +1,55 @@
+import { SVGPathData } from '../SVGPathData.js';
+import { SVGPathDataTransformer } from '../index.js';
+import { arePointsCollinear, type Point } from '../mathUtils.js';
+import type { SVGCommand } from '../types.js';
+
+/**
+ * Process a path and remove collinear points
+ * @param commands Array of SVG path commands to process (must be absolute)
+ * @returns New array with collinear points removed
+ */
+export function REMOVE_COLLINEAR(commands: SVGCommand[]): SVGCommand[] {
+  if (commands.length <= 2) return commands; // exit early if there are less than 3 points
+
+  const results: SVGCommand[] = [];
+
+  const points: Point[] = commands.map(
+    SVGPathDataTransformer.INFO((cmd, pXAbs, pYAbs) => {
+      // Calculate absolute coordinates and normlise HV
+      const isRelatve = 'relative' in cmd && cmd.relative;
+      return [
+        'x' in cmd ? cmd.x + (isRelatve ? pXAbs : 0) : pXAbs,
+        'y' in cmd ? cmd.y + (isRelatve ? pYAbs : 0) : pYAbs,
+      ];
+    }),
+  );
```

Comment:
i was thinking about using TO_ABS instead but i would also need to normalize HV, and this way way easier